azkaban-common/src/main/java/azkaban/flow/CommonJobProperties.java

```diff
@@ -140,6 +140,10 @@
    */
   public static final String SUBMIT_USER = "azkaban.flow.submituser";
 
+  /**
+   * Represents whether the flow is a scheduled flow or an adhoc execution
```

Comment:
Could you also mention the "event triggered" source?

Reply:
Done